assignment.js

```diff
@@ -112,6 +122,26 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 3:')
 
 // Add your code below this line
+let playerAChoice = ""
```

Comment:
🛠 Read the comment below and update your code accordingly

@jimernest90 Looks like your logic is correct but the syntax for your test conditions is incorrect

Here you are trying to test if playerA has selected 'rock' and playerB has selected:
`playerAChoice === "rock", playerBChoice ==="paper"`

you need to use a logical AND to make this work: 

`playerAChoice === "rock" && playerBChoice ==="paper"`

[Check out the class slide on logical operators for more info](https://jsd20190730.github.io/02-javascript-basics/logical-operators)

Please update your code accordingly